app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -0,0 +1,89 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
+    <div class="inset">
+      <p>Becoming a secondary school teacher is a great career choice if you want to make a difference in young people's lives. As students progress through school, you will directly influence their future prospects through the grades you help them achieve and the values you share with them.</p>
+      <p>As a specialist, you'll inspire students to develop a passion for your subject. You'll give them the skills and knowledge needed to tackle increasingly complex content and motivate some of them to pursue your subject beyond school.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+      <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_art_richard' %>
+    </section>
+    <h2 class="heading--box-blue">Key stages</h2>
```

Comment:
As discussed, might be worth changing title to what you'll teach and then moving the content from the what you'll teach section to here, before the key stage bits. Might need to reduce the amount of content in the what you'll teach section.

Also, I'd maybe remove the bit about 'teach any other subject' from the partial in the what you'll teach section, not sure it works when talking about secondary as it's not a specific subject.